### PR TITLE
Integrate other forks

### PR DESCRIPTION
**Pull request by groma84**
I consolidated other forks into one and successfully tested it on an Raspberry Pi 4

### CODE REVIEW

File: mix.lock [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
   "earmark": {:hex, :earmark, "1.2.4", "99b637c62a4d65a20a9fb674b8cffb8baa771c04605a80c911c4418c69b75439", [:mix], [], "hexpm"},
-  "elixir_make": {:hex, :elixir_make, "0.4.2", "332c649d08c18bc1ecc73b1befc68c647136de4f340b548844efc796405743bf", [:mix], [], "hexpm"},
-  "ex_doc": {:hex, :ex_doc, "0.18.3", "f4b0e4a2ec6f333dccf761838a4b253d75e11f714b85ae271c9ae361367897b7", [:mix], [{:earmark, "~> 1.1", [hex: :earmark, repo: "hexpm", optional: false]}], "hexpm"},
-  "excoveralls": {:hex, :excoveralls, "0.9.1", "14fd20fac51ab98d8e79615814cc9811888d2d7b28e85aa90ff2e30dcf3191d6", [:mix], [{:hackney, ">= 0.12.0", [hex: :hackney, repo: "hexpm", optional: false]}, {:jason, "~> 1.0", [hex: :jason, repo: "hexpm", optional: false]}], "hexpm"},
+  "earmark_parser": {:hex, :earmark_parser, "1.4.12", "b245e875ec0a311a342320da0551da407d9d2b65d98f7a9597ae078615af3449", [:mix], [], "hexpm", "711e2cc4d64abb7d566d43f54b78f7dc129308a63bc103fbd88550d2174b3160"},
+  "elixir_make": {:hex, :elixir_make, "0.6.2", "7dffacd77dec4c37b39af867cedaabb0b59f6a871f89722c25b28fcd4bd70530", [:mix], [], "hexpm", "03e49eadda22526a7e5279d53321d1cced6552f344ba4e03e619063de75348d9"},
```
Comment (visciang) [inline mix.lock:6]: There are some old unused deps in the lock file.
"mix deps.clean --unlock --unused" will clean them

**Review by visciang — CHANGES_REQUESTED** [review]
Thank you for your contribution!
Could you address the points I've commented in the PR, please?

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # App artifacts
+.vscode
```
[the remark this answers is not in the document]
Comment (groma84) [reply .gitignore:2]: I didn't even know there was a global gitignore file! Thanks for the info.

File: .gitignore [hunk 3]
```diff
@@ -14,3 +15,5 @@
 
 # Generate on crash by the VM
 erl_crash.dump
+
+/priv
```
[the remark this answers is not in the document]
Comment (groma84) [reply .gitignore:19]: But there is a build artifact placed into the folder ("dht") - this has to be ignored at least.

**Comment by groma84** [comment]
Thanks for the feedback (and the awesome library)!
I pushed a new commit where I tried to integrate all requested changes. Please review again.

**Review by visciang — APPROVED** [review]

**Pull request merged**